<?xml version='1.1' encoding='UTF-8'?>
<Tag plugin="workflow-support@839.v35e2736cfd5c">
  <node class="cps.n.StepEndNode" plugin="workflow-cps@3520.va_8fc49e2f96f">
    <parentIds>
      <string>36</string>
    </parentIds>
    <id>37</id>
    <startId>3</startId>
  </node>
  <actions>
    <wf.a.TimingAction plugin="workflow-api@1200.v8005c684b_a_c6">
      <startTime>1667831316999</startTime>
    </wf.a.TimingAction>
    <wf.a.ErrorAction plugin="workflow-api@1200.v8005c684b_a_c6">
      <error class="hudson.AbortException">
        <detailMessage>No credentials found for id &quot;kubernetes&quot;</detailMessage>
        <stackTrace>
          <trace>org.csanchez.jenkins.plugins.kubernetes.KubectlBuildWrapper.setUp(KubectlBuildWrapper.java:92)</trace>
          <trace>org.jenkinsci.plugins.workflow.steps.CoreWrapperStep$Execution2.doStart(CoreWrapperStep.java:121)</trace>
          <trace>org.jenkinsci.plugins.workflow.steps.GeneralNonBlockingStepExecution.lambda$run$0(GeneralNonBlockingStepExecution.java:77)</trace>
          <trace>java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)</trace>
          <trace>java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)</trace>
          <trace>java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)</trace>
          <trace>java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)</trace>
          <trace>java.base/java.lang.Thread.run(Thread.java:829)</trace>
        </stackTrace>
        <suppressedExceptions class="empty-list"/>
      </error>
    </wf.a.ErrorAction>
    <wf.a.ErrorAction plugin="workflow-api@1200.v8005c684b_a_c6">
      <error class="hudson.AbortException" reference="../../wf.a.ErrorAction/error"/>
    </wf.a.ErrorAction>
  </actions>
</Tag>